controllers/vrg_volrep.go

```diff
@@ -1934,8 +1937,6 @@ func (v *VRGInstance) restorePVsFromObjectStore(objectStore ObjectStorer, s3Prof
 func (v *VRGInstance) restorePVCsFromObjectStore(objectStore ObjectStorer, s3ProfileName string) (int, error) {
 	pvcList, err := downloadPVCs(objectStore, v.s3KeyPrefix())
 	if err != nil {
-		v.log.Error(err, fmt.Sprintf("error fetching PVC cluster data from S3 profile %s", s3ProfileName))
-
 		return 0, err
```

Comment:
Same, context for the error may help.

Maybe we get good enough errors - the best way to tell is to simulate failure and look at the logs with the errors.

Reply:
Adding context in the caller function